zookeeper-jute/src/main/java/org/apache/jute/compiler/JRecord.java

```diff
@@ -767,4 +790,53 @@ public static String getCsharpFQName(String name) {
         }
         return fQName.toString();
     }
+
+    private String getJavaFieldComments(JField jField) {
+        StringJoiner javaFieldCommentsJoiner =
+                new StringJoiner("\n   * ", "  /**\n   * ", "\n   */\n").setEmptyValue("");
+        return getFieldComments(jField, javaFieldCommentsJoiner);
+    }
+
+    private String getCFieldComments(JField jField) {
+        StringJoiner cFieldCommentsJoiner =
+                new StringJoiner("\n     * ", "    /**\n     * ", "\n     */\n").setEmptyValue("");
+        return getFieldComments(jField, cFieldCommentsJoiner);
+    }
+
+    private String getFieldComments(JField jField, StringJoiner fieldCommentsJoiner) {
+        List<String> fieldSpecialToken = jCommentGenerator.getFieldCommentsList(jField);
+        return getComments(fieldCommentsJoiner, fieldSpecialToken);
+    }
+
+    private String getRecordComments() {
+        List<String> classSpecialToken = jCommentGenerator.getRecordCommentsList(this);
+
+        StringJoiner recordCommentsJoiner =
+                new StringJoiner("\n * ", "/**\n * ", "\n */\n").setEmptyValue("");
+        return getComments(recordCommentsJoiner, classSpecialToken);
+    }
+
+    private static String getComments(StringJoiner commentsJoiner, List<String> commentList) {
+        if (commentList == null || commentList.isEmpty()) {
+            return commentsJoiner.toString();
+        }
+
+        for (String commentMulti : commentList) {
+            if (commentMulti == null || commentMulti.isEmpty()) {
+                continue;
+            }
+
+            String[] commentArray = commentMulti.split("\n");
+            for (String comment : commentArray) {
+
+                comment = comment.replaceAll("//|/\\*\\*|/\\*|\\*\\*/|\\*/", "")
```

Comment:
I do think people will want to maintain this kind of code.

Reply:
s/do/don't/

Reply:
> I do think people will want to maintain this kind of code.
I'll amend that and I won't touch the comments so they're not needed here.